ablySpec/RealtimeClientConnection.swift

```diff
@@ -1302,6 +1302,54 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15d
+                pending("should recover from disconnection and messages delivered once the connection is resumed") {
```

Comment:
Can we change this to "should recover from disconnection and publish queued messages once the connection is resumed"?